tokio-timer/src/lib.rs

```diff
@@ -59,9 +59,14 @@ pub use timer::{set_default, Timer};
 
 use std::time::{Duration, Instant};
 
+/// Create a Future that completes at `deadline`.
+pub fn delay(deadline: Instant) -> Delay {
+    Delay::new(deadline)
+}
+
 /// Create a Future that completes in `duration` from now.
 pub fn sleep(duration: Duration) -> Delay {
```

Comment:
Not for this PR, but it looks like `sleep` and `delay` do the same thing now... I added a note to #1261 to figure this out. My inclination is to remove `sleep` in favor of `delay`.

Reply:
It's not exactly the same, `delay` will wake at a specified time, `sleep` will wake at `now` + specified duration. In my experience `sleep` is actually much more useful, since usually I want to asynchronously "sleep" for some time from `now`, rather than asynchronously wake up at some specific moment in the future. Maybe the names of the functions may be changed, but `sleep` sounds reasonable to me, as it is an async counterpart to an actual `thread::sleep`.

Reply:
ok, we can keep both for now